src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -24,14 +25,40 @@ public static class Base64Encoding
         /// </summary>
         /// <param name="value">TBD</param>
         /// <returns>TBD</returns>
-        public static string Base64Encode(this long value) => Base64Encode(value, new StringBuilder()).ToString();
+        public static string Base64Encode(this long value)
+        {
+            return Base64Encode(value, "");
+        }
+
+        internal static string Base64Encode(this long value, string prefix)
+        {
+            // 11 is the number of characters it takes to represent long.MaxValue
+            // so we will never need a larger size for encoding longs
+            Span<char> sb = stackalloc char[11 + prefix?.Length ?? 0];
+            var spanIndex = 0;
+            if (!string.IsNullOrWhiteSpace(prefix) && prefix.Length > 0)
+            {
+                prefix.AsSpan().CopyTo(sb);
+                spanIndex = prefix.Length;
+            }
+
+            var next = value;
+            do
+            {
+                var index = (int)(next & 63);
+                sb[spanIndex++] = Base64Chars[index];
+                next = next >> 6;
+            } while (next != 0);
+            return sb.Slice(0, spanIndex).ToString();
+        }
 
         /// <summary>
         /// TBD
         /// </summary>
         /// <param name="value">TBD</param>
         /// <param name="sb">TBD</param>
         /// <returns>TBD</returns>
+        [Obsolete("Do not use. Pass in prefix as a string instead.")]
```

Comment:
Still useful since this method really shouldn't have been exposed as `public` anyway, IMHO

Reply:
We had a bad habit in the very early days of Akka.NET making a lot of the internals public "in case someone wanted to play with them!" and we've been gradually undoing that in our minor version releases....